packages/merge-styles/src/IRawStyleBase.ts

```diff
@@ -1267,11 +1267,39 @@ export interface IRawStyleBase extends IRawFontStyle {
    */
   margin?: ICSSRule | ICSSPixelUnitRule;
 
+  /**
```

Comment:
Can you update the comment to align with the descriptions for `marginInlineEnd` and `marginInlineStart` (as well as the `padding*` rules). Ex:

> The margin-block-end CSS property defines the logical block end margin of an element, which maps to a physical margin depending on the element's writing mode, directionality, and text orientation. It corresponds to the margin-top, margin-right, margin-bottom or margin-left property depending on the values defined for writing-mode, direction, and text-orientation.

Reply:
Thanks, I've updated them. I'd copied them from e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-end, which has different styles for different properties.